release/models/gnpsi/openconfig-gnpsi.yang

```diff
@@ -0,0 +1,70 @@
+module openconfig-gnpsi {
+  yang-version "1";
+  namespace "http://openconfig.net/yang/gnpsi";
+  prefix "oc-gnpsi";
+
+  import openconfig-extensions {
+    prefix oc-ext;
+  }
+
+  import openconfig-system {
+    prefix oc-sys;
+  }
+
+  import openconfig-system-grpc {
+    prefix oc-sys-grpc;
+  }
+
+  import openconfig-gnpsi-types {
+    prefix oc-gnpsit;
+  }
+
+  import openconfig-yang-types {
+    prefix oc-yang;
+  }
+
+  organization
+    "OpenConfig Working group";
+  contact
+    "www.openconfig.net";
+
+  description
+    "This module defines configuration and operational state data
+    related to gRPC Network Packet Sampling Interface (gNPSI).
+
+    GNPSI Proto Reference: https://github.com/openconfig/gnpsi";
+
+  oc-ext:openconfig-version "0.1.0";
+  oc-ext:catalog-organization "openconfig";
+  oc-ext:origin "openconfig";
+
+  revision 2024-05-16 {
+    description
+      "Initial revision.";
+    reference "0.1.0";
+  }
+
+  grouping gnpsi-counters-top {
+    description
+      "Top-level container of operational data for gNPSI counters.";
+
+    leaf packets-error {
+      type oc-yang:counter64;
+      description
+        "Error packets/datagram failed to be sent out by gNPSI
+        service for a specific client.";
+    }
+  }
```

Comment:
Reading the [gnpsi server reference implementation](https://github.com/search?q=repo%3Aopenconfig%2Fgnpsi%20IncrementWriteErrorCount()&type=code), I suggest this name and description.  

```suggestion
leaf sample-send-error {
      type oc-yang:counter64;
      description
        "A count of errors the gnpsi server encountered when sending a packet sample to a grpc client.";
    }
  }
```